app/decorators/podcast_episode_decorator.rb

```diff
@@ -8,4 +8,12 @@ def comments_to_show_count
   def cached_tag_list_array
     (tag_list || "").split(", ")
   end
+
+  def readable_publish_date
+    published_at&.strftime("%b %e")
```

Comment:
I think this should display the year for older podcasts, see how for example, it is done for the comment:

```ruby
  def readable_publish_date
    if created_at.year == Time.current.year
      created_at.strftime("%b %e")
    else
      created_at.strftime("%b %e '%y")
    end
  end
```